resources/web/README.md

```diff
@@ -1,15 +1,69 @@
+---    
+sidebar: auto
+---      
+
 # Web Development
```

Comment:
A lot of the resources you've listed are text based. I would encourage you to search for playlists and videos such as these: 

https://www.youtube.com/watch?v=guvsH5OFizE&t=1s
 https://www.youtube.com/playlist?list=PLo3w8EB99pqLEopnunz-dOOBJ8t-Wgt2g